pkgs/development/libraries/wxwidgets/2.8/default.nix

```diff
@@ -22,6 +22,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig ];
 
+  patches = [
+   ./patches/fix_assertion_using_hide_in_destroy.diff
```

Comment:
The patch should not be placed into the repository but fetched using `fetchurl`.